### PR TITLE
Add plantuml 1.2021.7

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Notes:
- much copied from @conda-forge/antlr (especially the wrapper scripts)

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-java ready for review!

File: recipes/plantuml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  url: https://github.com/plantuml/plantuml/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a8e3209171ccc57a9be2a267b2bcea976cc94a535c143ee4a77ba4dab5d71698
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - maven
+    - graphviz
```
Comment (chrisburr) [inline recipes/plantuml/meta.yaml:17]: I don't really know much about Java but I would expect this to be needed in `host` rather than `build`.

Comment (bollwyvl) [reply recipes/plantuml/meta.yaml:17]: i'll give it a shot!

Comment (bollwyvl) [reply recipes/plantuml/meta.yaml:18]: linux did not like that, will put it back!

Comment (bollwyvl) [reply recipes/plantuml/meta.yaml:17]: back to green....

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes ping!

**Comment by zklaus** [comment]
Looks like this one just fell through the cracks.

@conda-forge-admin, please ping team

**Comment by bollwyvl** [comment]
Yeah, this version would be fine, could also rev it to latest if desired... or just wait to do so on eventual feedstock.

**Comment by carterbox** [comment]
@bollwyvl Ping me next Friday if @mariusvniekerk hasn't reviewed by then.

**Comment by zklaus** [comment]
Hm. I'm not @bollwyvl, but hopefully it's still ok for me to ping @bollwyvl, @mariusvniekerk, @carterbox.

**Pull request merged**